### PR TITLE
feat: support responsive grid

### LINKED ISSUE #5: Feature request: Responsive grid
Sometimes the grid changes on different breakpoints. It would be great to have that option in the parameters too.
Great work BTW.

### PR DESCRIPTION
**Pull request by thearnica**
Was looking at tremendous work happening at https://github.com/maraisr/storybook-addon-grid/pull/11 and probably there are too many changes.

Let's keep in mind why we want responsive grid - only because the target application needs it.

So, let's ask yourself a question - and how the target application gonna do it? How it will provide different grid settings for different media points?

The answer is "somehow", and we want to tap-into that somehow. Be a part of it, not replicate in any way.

-------

Long story short - embrace css variables and let application "CSS configuration" let us know what to do.

```tsx
Example.parameters = {
    grid: {
        columns: 'var(--columns)',
        gutter: 'var(--gutter)',
        gap: 'var(--gap)',
    }
}
```

> PS: gutters and gap already works this way. Now `columns` can use css var.

fixes #5 
closes #11

### CODE REVIEW

File: src/components/ui.tsx [hunk 1]
```diff
@@ -88,17 +88,21 @@ export const Grids: FunctionComponent<
 	color = 'rgba(255, 0, 0, 0.1)',
 	gutter = '50px',
 	maxWidth = '1024px',
+	maxColumns = 24
 }) => {
+	const numberOfColumns = typeof(columns) ==='number' ? columns : maxColumns;
+	const grid = typeof(columns) ==='number' ? `repeat(${columns}, 1fr)` : `repeat(var(${columns.match(/var\((.*)\)/)![1]}), 1fr)`
```
Comment (maraisr) [inline src/components/ui.tsx:94]: Why are we matching here?
